pom.xml

```diff
@@ -13,15 +13,12 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>21</maven.compiler.source>
-        <maven.compiler.target>21</maven.compiler.target>
-        <maven.compiler.compilerVersion>21</maven.compiler.compilerVersion>
         <aws.java.sdk.version>2.23.12</aws.java.sdk.version>
-        <junit5.version>5.9.1</junit5.version>
-        <log4j.version>2.22.1</log4j.version>
+        <junit5.version>5.10.1</junit5.version>
```

Comment:
Since this will probably only be used in one place (the JUnit 5 BOM declaration), this could probably just be inlined.